scripts/install/toran.sh

```diff
@@ -76,7 +76,7 @@ elif [ "${TORAN_CRON_TIMER}" == "fifteen" ]; then
 elif [ "${TORAN_CRON_TIMER}" == "half" ]; then
     CRON_TIMER="*/30 * * * *"
 elif [ "${TORAN_CRON_TIMER}" == "hour" ]; then
-    CRON_TIMER="*/1 * * * *"
+    CRON_TIMER="* 1 * * *"
```

Comment:
Correct value to execute cron every hours (*h:00) should be : `0 * * * *` . No ?